core/src/main/java/org/apache/brooklyn/core/location/access/PortForwardManagerImpl.java

```diff
@@ -83,9 +83,6 @@
 
     private final Map<AssociationListener, Predicate<? super AssociationMetadata>> associationListeners = new ConcurrentHashMap<AssociationListener, Predicate<? super AssociationMetadata>>();
 
-    @Deprecated
-    protected final Map<String,String> publicIpIdToHostname = new LinkedHashMap<String,String>();
```

Comment:
Is there a strategy for removing deprecated fields short of making them private and leaving them forever? Not sure we can do anything sensible to transform its data during rebind.